webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue

```diff
@@ -139,13 +141,19 @@ export default {
     },
     toolbarActionsDisplay() {
       return this.userCanPost && this.spaceId;
+    },
+    filterIconColor() {
+      return this.streamFilter !== 'all_stream' && 'primary';
     }
   },
   created() {
     if (!this.user) {
       this.$userService.getUser(eXo.env.portal.userName)
         .then(user => this.user = user);
     }
+    document.addEventListener('activity-stream-type-filter-changed', event => {
```

Comment:
Can you use the existing event `activity-stream-type-filter-applied` ?

```suggestion
    document.addEventListener('activity-stream-type-filter-applied', event => {
```

Reply:
done!